content/blog/2019-12-mio-v0.7-alpha.1.md

```diff
@@ -0,0 +1,197 @@
++++
+date = "2019-12-13"
+title = "Announcing Mio 0.7-alpha.1"
+description = "December 13th, 2019"
+menu = "blog"
+weight = 984
++++
+
+Mio 0.7 is the work of [various contributors] over the course of roughly half a
+year. Compared to Mio version 0.6 version 0.7 reduces the size of the provided
+API in an attempt to simply the implementation and usage. The API version 0.7
+will be close to the proposed API for a future version 1.0. The scope for the
+crate was reduced to providing a cross-platform event notification mechanism and
+commonly used types such as cross-thread poll waking and non-blocking networking
+I/O primitives.
+
+[various contributors]: https://github.com/tokio-rs/mio/graphs/contributors?from=2019-03-01&to=2019-12-13
+
+
+## Major changes
+
+Since this is a large release only some highlights are described here, all
+changes can be found in the [change log]. Overall a large number of API changes
+have been made to reduce the complexity of the implementation and remove
+overhead were possible.
+
+[change log]: https://github.com/tokio-rs/mio/blob/master/CHANGELOG.md#070-alpha1
+
+
+### Wrapping native OS types
+
+In version 0.6, Mio defined its own `Event` structure to which the native OS
+types where converted. This meant that after polling the native types (i.e.
+`kevent` or `epoll_event` structures) would be converted into Mio's `Event`
+type. In version 0.7, the `Event` was changed to simply be a type wrapper around
+`kevent` or `epoll_event`, which provides convenience methods to get the event
+readiness indicators (such as readable or writable). Similar changes have been
+made throughout the crate. For example `Poll` is now just a file descriptor on
+Unix.
+
+The way events are handled using the `Events` struct has also changed. In
+version 0.6.10, index access was already deprecated and replaced with an
+iteration API. The index access was completely removed in version 0.7, and the
+iteration API was changed to return a reference to `Event` rather then making a
+copy, but since all methods on `Event` only need a reference, this shouldn't be
```

Comment:
Bit of a long sentence going on here (and typo: `then` -> `than`).

How about: `.. rather than making copy. All methods on 'Event' only need a reference, so this shouldn't ..`